.github/workflows/publish_package.yml

```diff
@@ -36,6 +36,9 @@ jobs:
         PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
       run: |
         ./publish.sh "https://upload.pypi.org/legacy/" "$PYPI_TOKEN"
+    - name: Generate Conda Metadata
+      run: |
+        ./generate_conda_meta.sh
```

Comment:
this will put the metadata file in the conda sub dir?

Reply:
@matt-caz 
as in the error was 
meta.yaml must exist in the directory that is being packaged and published.

Reply:
Yes, that's what this step does.